manifests/backend.pp

```diff
@@ -343,13 +343,12 @@
 
   exec { 'sensu-backend init':
     path        => '/usr/bin:/bin:/usr/sbin:/sbin',
-    refreshonly => true,
     environment => [
       'SENSU_BACKEND_CLUSTER_ADMIN_USERNAME=admin',
       "SENSU_BACKEND_CLUSTER_ADMIN_PASSWORD=${sensu::password}",
     ],
     returns     => [0, 3],
-    subscribe   => Package['sensu-go-backend'],
+    unless      => 'sensu-backend init ; [ $? -eq 3 ] && exit 0 || exit 1',
```

Comment:
Given the complexity of inline shell code and an odd exit code, please add a comment above this resource to explain what is happening and why.

Reply:
Added comments.